.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,49 @@
+# Publish archives to PyPI and TestPyPI using GitHub Actions
+name: Publish to PyPI
+
+# Only run for tagged releases and pushes to the development branch
+on:
+  release:
+    types:
+      - published
+  push:
+    branches:
+      - class-redesign
```

Comment:
Is this here only to test during development? Otherwise, do you expect a PyPI build (test or otherwise) to be initiated from this branch?